libbeat/cmd/instance/beat.go

```diff
@@ -450,7 +450,7 @@ func (b *Beat) loadMeta() error {
 	}
 
 	metaPath := paths.Resolve(paths.Data, "meta.json")
-	logp.Info("Beat metadata path: %v", metaPath)
+	logp.Debug("beat", "Beat metadata path: %v", metaPath)
```

Comment:
Are you sure you want to also move this one to `Debug`?

Reply:
It seemed to me like a bit redundant because we already print the data path (where the meta file is located) and the UUID (which is the contents of the meta file). But happy to add it back if you think it's worth.

Reply:
SGTM. Did miss the part that we already log the data path. That is enough.